core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -714,6 +714,8 @@ static function ($allowedTag)
 
 				// Request token
 				case 'request_token':
+					trigger_deprecation('contao/core-bundle', '4.13', 'Using the request_token insert tag has been deprecated and will no longer work in Contao 5.0.');
+
 					$arrCache[$strTag] = REQUEST_TOKEN;
```

Comment:
Should probably also just take the service so we can get rid of all the usages of the constants as well?

Reply:
The `REQUEST_TOKEN` constant is currently used 100+ times in the code base. Let's refactor this in another PR, wdyt?